<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_SimplePaintClass</class>
 <widget class="QMainWindow" name="Qt_SimplePaintClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt_SimplePaint</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="Scene" name="SceneWidget_placeholder" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>811</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>6</y>
      <width>808</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="RectButton">
         <property name="text">
          <string>Rectangle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CircleButton">
         <property name="text">
          <string>Circle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="TriangleButton">
         <property name="text">
          <string>Triangle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PathButton">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SpiralButton">
         <property name="text">
          <string>Spiral</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SegmentButton">
         <property name="text">
          <string>Segment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="TextItemButton">
         <property name="text">
          <string>TextItem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ComplexItemButton">
         <property name="text">
          <string>Complex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MoveButton">
         <property name="text">
          <string>Move</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="Check_Filled">
         <property name="text">
          <string>Filled</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Scene</class>
   <extends>QWidget</extends>
   <header>Scene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Qt_SimplePaint.qrc"/>
 </resources>
 <connections/>
</ui>
